### ISSUE TITLE
Remove the size limit of picture in picture video popout

### ISSUE THREAD

**Issue by wkoncki** [electron/electron#21654]
* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

### Problem Description
Electron 7 enabled the ability to toggle picture in picture (pip) mode for videos. It works great but the resulting popout has a a maximum size limitation (around 1/3 of screen). I think that this limitation impacts potential use of this functionality. For instance you cannot move video to another screen and use it's full size.

### Proposed Solution
Remove the maximum size limitation and allow the user to set any video size.

**Comment by cloewen8** [comment]
I agree it should not be limited, but for different reasons. In some cases it may need to be expanded temporarily to see small-print.

**Comment by wkoncki** [comment]
@cloewen8 That a very good point actually. Bigger popout will prove more useful for people without good vision, and when video contains small text. In our case we show sport competitions and current popout does not allow for readable player numbers. That was pretty much a motivation for this issue.

**Comment by manhnt9** [comment]
We also need this 👍

**Comment by codebytere** [comment]
Hi, while we appreciate this enhancement request it's not something that our team will be able to implement in the near future, and since we use our issues tab as the team's backlog, we prefer to only retain issues that are on our immediate roadmap. If you'd like to make a PR for this request we would welcome that but in the meantime I'm going to go ahead and close this.
